files/jsondata/GroupData.json

```
@@ -1303,7 +1303,8 @@
         "Report",
         "ReportBody",
         "ReportError",
```

```suggestion
        "reportError",
```

Just while we are at it...